src/v0/destinations/hs/HSTransform-v2.js

```diff
@@ -235,10 +236,14 @@ const processTrack = async (message, destination) => {
 
 const batchIdentify = (arrayChunksIdentify, batchedResponseList, batchOperation) => {
   // list of chunks [ [..], [..] ]
+  const { destinationId } = arrayChunksIdentify[0][0].destination;
```

Comment:
```suggestion
  const { destinationId } = arrayChunksIdentify[0]?.[0]?.destination;
```

Reply:
optional chaning is not required here as we are itself building ensuring we have these indices